.github/stale.yml

```diff
@@ -0,0 +1,54 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 30
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
```

Comment:
are there any labels we want to add here, that we never want to be marked as stale? maybe infrastructure? other ones?

Reply:
Perhaps we could make a custom label that we could use for issues that need to sit around for a while.  I don't know what it might be called...